src/modules/endpoint/sagas/loadLanguages.js

```diff
@@ -24,7 +25,10 @@ export default function * loadLanguages (
 
   console.debug('Fetching languages')
 
-  const payload = yield call(() => createLanguagesEndpoint(baseUrl).request({ city }))
+  const appSettings = new AppSettings()
+  const apiUrlOverride = yield call(appSettings.loadApiUrlOverride)
+
+  const payload = yield call(() => createLanguagesEndpoint(apiUrlOverride || baseUrl).request({ city }))
```

Comment:
Those two cases `apiUrlOverride` is null or not should be tested in `loadLanguages.spec.js`.